shanghai/connection.py

```diff
@@ -38,7 +39,7 @@ def close(self):
             while not reader.at_eof():
                 line = await reader.readline()
                 line = line.strip()
-                current_logger.debug(">", line)
+                current_logger.info(">", line)
```

Comment:
I actually prefer those to be `debug`, because we get *a lot* of those. Outgoing messages on the other hand can stay `info`.